newrelic-api-client/src/main/java/com/ocadotechnology/newrelic/apiclient/model/dashboards/widget/WidgetPresentationThreshold.java

```diff
@@ -0,0 +1,16 @@
+package com.ocadotechnology.newrelic.apiclient.model.dashboards.widget;
```

Comment:
How is this class used?

Reply:
It had to be a leftover from the previous commit. I have rebased changes and `WidgetPresentationThreshold` is no longer in the changes.